sdk/include/xdk/exfuncs.h

```diff
@@ -785,6 +785,24 @@ ExFreePoolWithTag(
   _Pre_notnull_ __drv_freesMem(Mem) PVOID P,
   _In_ ULONG Tag);
 
+static inline
+PVOID NTAPI
+ExReAllocatePoolWithTag(
+    POOL_TYPE PoolType,
+    PVOID pOld,
+    SIZE_T cbOld,
+    SIZE_T cbNew,
+    ULONG Tag)
+{
+    PVOID pNew = ExAllocatePoolWithTag(PoolType, cbNew, Tag);
+    if (!pNew)
+        return NULL;
+
+    RtlCopyMemory(pNew, pOld, ((cbOld < cbNew) ? cbOld : cbNew));
+    ExFreePoolWithTag(pOld, Tag);
+    return pNew;
+}
+
```

Comment:
MS headers do not have such kind of function exported nor it has ever existed anywhere else so please revert this and implement it something like `IntReAllocatePool` or whatever since it seem it's exclusively for NTUSER here.

Reply:
Reverted and implemented as `IntReAllocatePoolWithTag`.